### PR TITLE
Update get-started.md

### PR DESCRIPTION
**Pull request by sarmentof**
Modifications made for adaptation to pt-BR translation.

### CODE REVIEW

File: content/react/pt/get-started.md [hunk 1]
```diff
@@ -6,20 +6,20 @@ description: "Configuração do React Storybook no ambiente de desenvolvimento"
 
 # Introdução
 
-Storybook funciona em paralelo á aplicação em modo de desenvolvimento.
-Ajuda na construção de componentes de interface de utilizador isolados de qualquer lógica e contexto da aplicação.
-Esta edição de Aprendizagem de Storybook é destinada para React.
-Encontram-se disponíveis outras edições quer para [Vue](/vue/pt/get-started), quer para [Angular](/angular/pt/get-started).
+Storybook funciona em paralelo à aplicação em modo de desenvolvimento.
+Ajuda na construção de componentes de interface de usuário, isolados de qualquer lógica e contexto da aplicação.
```
Comment (jonniebigodes) [inline content/react/pt/get-started.md]: In this case "Utilizador" or "usuário" is the same same thing. Both portuguese, either portuguese or brazilian portuguese speaking people do understand the meaning of the word.

File: content/react/pt/get-started.md [hunk 2]
```diff
@@ -6,20 +6,20 @@ description: "Configuração do React Storybook no ambiente de desenvolvimento"
...
+Esta edição de Aprenda Storybook é destinada para React.
+Encontram-se disponíveis outras edições tanto para [Vue](/vue/pt/get-started), quanto para [Angular](/angular/pt/get-started).
```
Comment (jonniebigodes) [inline content/react/pt/get-started.md]: In this case using either phrasing is correct, there's no need to change it.

File: content/react/pt/get-started.md [hunk 3]
```diff
@@ -6,20 +6,20 @@ description: "Configuração do React Storybook no ambiente de desenvolvimento"
...
 
 ![Storybook e a aplicação](/storybook-relationship.jpg)
 
-## Configuração de Storybook com React
+## Configurando o Storybook com React
 
-Irão ser necessárias algumas etapas adicionais de forma a ser possível configurar o processo de compilação no nosso ambiente de desenvolvimento.
-Para começar queremos usar o pacote [Create React App](https://github.com/facebook/create-react-app) ou como é vulgarmente conhecido (CRA), para configurar o nosso ambiente local e ativar o modo de testes com [Storybook](https://storybook.js.org/) e 
-[Jest](https://facebook.github.io/jest/) na nossa aplicação.
+Serão necessárias algumas etapas adicionais para que seja possível configurar o processo de compilação em nosso ambiente de desenvolvimento.
```
Comment (jonniebigodes) [inline content/react/pt/get-started.md]: I agree with almost of the change in here, but replacing "no" with "em", it's unnecessary, this will   transform it into a specified version. And not what's intended, it's portuguese in all it's form. Not a specification to pt-BR

File: content/react/pt/get-started.md [hunk 4]
```diff
@@ -44,34 +44,34 @@ yarn start
 ```
 
 <div class="aside">
-  Nota: Se <code>yarn test</code> emitir um erro, será necessário instalar o pacote <code>watchman</code>, tal como recomendado neste <a href="https://github.com/facebook/create-react-app/issues/871#issuecomment-252297884">problema</a>
+  Nota: Se o comando <code>yarn test</code> produzir um erro, será necessário instalar o pacote <code>watchman</code>, conforme recomendado nesta <a href="https://github.com/facebook/create-react-app/issues/871#issuecomment-252297884">thread</a>
```
Comment (jonniebigodes) [inline content/react/pt/get-started.md]: Why change the "problema" para "thread"? It's best to leave it as is. Both audiences can read it perfectly and understand the meaning. The rest of the phrase looks good.

File: content/react/pt/get-started.md [hunk 5]
```diff
@@ -44,34 +44,34 @@ yarn start
...
 </div>
 
 Sendo estes os seguintes: testes automáticos (Jest), desenvolvimento de componentes (Storybook) e a aplicação em si.
 
 ![3 modalidades](/app-three-modalities.png)
 
 
-Dependendo de qual parte da aplicação que estamos a trabalhar, podemos querer executar um ou mais simultâneamente.
-Visto que neste caso, o foco é a criação de um componente de interface de utilizador simples, iremos cingir-nos somente á execução de Storybook.
+Dependendo de qual parte da aplicação que estamos trabalhando, podemos querer executar um ou mais simultâneamente.
```
Comment (jonniebigodes) [inline content/react/pt/get-started.md]: Why the change to a diferent verb tenses? when the document was created it was done so that a language specificity was implemented. Trying to make as broader as possible

File: content/react/pt/get-started.md [hunk 6]
```diff
@@ -44,34 +44,34 @@ yarn start
...
+Visto que, neste caso, o foco é a criação de um componente de interface de usuário simples, continuaremos executando o Storybook.
```
Comment (jonniebigodes) [inline content/react/pt/get-started.md]: Once again "utilizador" or "usuario" it's the same thing. And trying to prevent a language specificity

File: content/react/pt/get-started.md [hunk 7]
```diff
@@ -44,34 +44,34 @@ yarn start
...
 
 ## Reutilização CSS
 
 A Taskbox reutiliza elementos de design do tutorial de React e GraphQL
-[Tutorial React e GraphQL](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858), como tal não será necessária a criação de CSS neste tutorial. 
+[Tutorial React e GraphQL](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858), por isso não será necessária a criação de CSS neste tutorial. 
 
-Com isto o conteúdo do ficheiro LESS será compilado num único ficheiro CSS e incluido na aplicação.
+Desta forma o conteúdo do arquivo LESS será compilado em um único arquivo CSS e incluído na aplicação.
```
Comment (jonniebigodes) [inline content/react/pt/get-started.md]: "Arquivo" and "ficheiro" are one and the same, "num" and "em um" are the one and the same, both readers can understand it, why the change? 
Thanks for the accent, missed that one

File: content/react/pt/get-started.md [hunk 8]
```diff
@@ -44,34 +44,34 @@ yarn start
...
 
-O CSS compilado encontra-se disponível [aqui](https://github.com/hichroma/learnstorybook-code/blob/master/src/index.css) e pela convenção CRA(Create React App) será necessário copiar seu conteúdo para o seguinte ficheiro src/index.css.
+O CSS compilado encontra-se disponível [aqui](https://github.com/hichroma/learnstorybook-code/blob/master/src/index.css) e pela convenção CRA(Create React App) será necessário copiar seu conteúdo para o seguinte arquivo src/index.css.
 
-![Interface Utilizador Taskbox](/ss-browserchrome-taskbox-learnstorybook.png)
+![Interface Usuário Taskbox](/ss-browserchrome-taskbox-learnstorybook.png)
 
 <div class="aside">
-  Se for necessária alguma alteração aos elementos de estilo, os ficheiros LESS originais encontram-se disponíveis no repositório Github.
+  Se desejar alterar o estilo, os arquivos LESS originais encontram-se disponíveis no repositório Github.
```
Comment (jonniebigodes) [inline content/react/pt/get-started.md]: Once again "arquivo" and "ficheiro" are one and the same and both readers can understand it perfectly, trying to maintain the document language agnostic as possible

File: content/react/pt/get-started.md [hunk 9]
```diff
@@ -44,34 +44,34 @@ yarn start
...
+[Tutorial React e GraphQL](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858), por isso não será necessária a criação de CSS neste tutorial. 
 
-Com isto o conteúdo do ficheiro LESS será compilado num único ficheiro CSS e incluido na aplicação.
+Desta forma o conteúdo do arquivo LESS será compilado em um único arquivo CSS e incluído na aplicação.
 
-O CSS compilado encontra-se disponível [aqui](https://github.com/hichroma/learnstorybook-code/blob/master/src/index.css) e pela convenção CRA(Create React App) será necessário copiar seu conteúdo para o seguinte ficheiro src/index.css.
+O CSS compilado encontra-se disponível [aqui](https://github.com/hichroma/learnstorybook-code/blob/master/src/index.css) e pela convenção CRA(Create React App) será necessário copiar seu conteúdo para o seguinte arquivo src/index.css.
 
-![Interface Utilizador Taskbox](/ss-browserchrome-taskbox-learnstorybook.png)
+![Interface Usuário Taskbox](/ss-browserchrome-taskbox-learnstorybook.png)
```
Comment (jonniebigodes) [inline content/react/pt/get-started.md]: Same as mentioned above, "Utilizador" and "Usuário" are one and the same both readers can understand it

File: content/react/pt/get-started.md [hunk 10]
```diff
@@ -44,34 +44,34 @@ yarn start
...
 
 A Taskbox reutiliza elementos de design do tutorial de React e GraphQL
-[Tutorial React e GraphQL](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858), como tal não será necessária a criação de CSS neste tutorial. 
+[Tutorial React e GraphQL](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858), por isso não será necessária a criação de CSS neste tutorial. 
 
-Com isto o conteúdo do ficheiro LESS será compilado num único ficheiro CSS e incluido na aplicação.
+Desta forma o conteúdo do arquivo LESS será compilado em um único arquivo CSS e incluído na aplicação.
 
-O CSS compilado encontra-se disponível [aqui](https://github.com/hichroma/learnstorybook-code/blob/master/src/index.css) e pela convenção CRA(Create React App) será necessário copiar seu conteúdo para o seguinte ficheiro src/index.css.
+O CSS compilado encontra-se disponível [aqui](https://github.com/hichroma/learnstorybook-code/blob/master/src/index.css) e pela convenção CRA(Create React App) será necessário copiar seu conteúdo para o seguinte arquivo src/index.css.
```
Comment (jonniebigodes) [inline content/react/pt/get-started.md]: "Ficheiro" and "Arquivo" are one and the same, no need for change in here, as both readers can understand it perfectly

**Comment by jonniebigodes** [comment]
i know i'm new here, but i would like to add my two cents in this matter. Left some revisions on the code so that the translation can be done in a uniform manner

File: content/react/pt/get-started.md [hunk 11]
```diff
@@ -44,34 +44,34 @@ yarn start
...
-  Se for necessária alguma alteração aos elementos de estilo, os ficheiros LESS originais encontram-se disponíveis no repositório Github.
+  Se desejar alterar o estilo, os arquivos LESS originais encontram-se disponíveis no repositório Github.
 </div>
 
 ## Adicionar recursos
 
-Irá ser necessário adicionar também as pastas com o tipo de letra e ícones que se encontram disponíveis [aqui](https://github.com/hichroma/learnstorybook-code/tree/master/public) á pasta `public`. Ao adicionar estes elementos, a aplicação irá renderizar de forma algo estranha. 
-Mas isto é de esperar, visto que não iremos trabalhar na aplicação agora.
+Será necessário adicionar também as pastas com o tipo de letra e icones que se encontram disponíveis [aqui](https://github.com/hichroma/learnstorybook-code/tree/master/public) à pasta `public`. Ao adicionar estes elementos, a aplicação irá renderizar de forma um pouco estranha. 
+Mas isto é esperado, visto que não iremos trabalhar na aplicação agora.
```
Comment (jonniebigodes) [inline content/react/pt/get-started.md]: Phrasing sounds pretty much perfectly clear for all. Also "ícones" is well written. There's an accent in there

File: content/react/pt/get-started.md (hunk 4 above)
Comment (sarmentof) [reply content/react/pt/get-started.md]: This link refers to yarn but is not a specific problem of it.
The main problem is about the npm and in the middle of the thread they talk about the yarn.

File: content/react/pt/get-started.md (hunk 1 above)
Comment (sarmentof) [reply content/react/pt/get-started.md]: Sorry, it is not common in Brazil to use/read "Utilizador".

File: content/react/pt/get-started.md (hunk 2 above)
Comment (sarmentof) [reply content/react/pt/get-started.md]: Sorry, it is not common in Brazil to use/read "Utilizador".

File: content/react/pt/get-started.md (hunk 6 above)
Comment (sarmentof) [reply content/react/pt/get-started.md]: Sorry, it is not common in Brazil to use/read "Utilizador".

File: content/react/pt/get-started.md (hunk 11 above)
Comment (sarmentof) [reply content/react/pt/get-started.md]: Sorry, it's just a grammatical correction related to the verbs "ir" and "ser".

File: content/react/pt/get-started.md (hunk 8 above)
Comment (sarmentof) [reply content/react/pt/get-started.md]: Sorry, "Ficheiro" is not a word used in Brazil.

File: content/react/pt/get-started.md (hunk 1 above)
Comment (jonniebigodes) [reply content/react/pt/get-started.md]: The word is not common, but it does exist and is recognized for both audiences.

File: content/react/pt/get-started.md (hunk 4 above)
Comment (jonniebigodes) [reply content/react/pt/get-started.md]: > The main problem is about the npm and in the middle of the thread they talk about the yarn.
This phrase exists, due to the fact that the guide is heavily influenced by Create React App and with that as they are much more prone to using yarn as opposed to npm, for reasons that are out of the scope of this guide. The writter left in that note. That sentence is being corrected to allow it both ways. Using npm or yarn

File: content/react/pt/get-started.md (hunk 8 above)
Comment (jonniebigodes) [reply content/react/pt/get-started.md]: The word is not not used, but it does exist and is recognized for both audiences in terms of it's meaning.

File: content/react/pt/get-started.md (hunk 9 above)
Comment (jonniebigodes) [reply content/react/pt/get-started.md]: The word is not common, but it does exist and is recognized for both audiences in terms of it's meaning.

File: content/react/pt/get-started.md (hunk 10 above)
Comment (jonniebigodes) [reply content/react/pt/get-started.md]: The word is not used, but it does exist and is recognized for both audiences in terms of it's meaning.

**Comment by domyen** [comment]
@sarmentof is this issue still active? We're updating the guide this month and want to clear out stuff that's no longer active. 🙏
